lib/db/benchmark_test.go

```diff
@@ -36,9 +40,9 @@ func init() {
 	oneFile = firstHalf[middle-1 : middle]
 
 	ldb, _ := tempDB()
-	s = db.NewFileSet("test)", fs.NewFilesystem(fs.FilesystemTypeBasic, "."), ldb)
-	replace(s, remoteDevice0, files)
-	replace(s, protocol.LocalDeviceID, firstHalf)
+	benchS = db.NewFileSet("test)", fs.NewFilesystem(fs.FilesystemTypeBasic, "."), ldb)
```

Comment:
Bracket in string

Reply:
Yeah, that's common to all (most?) lib/db tests - no idea why, I just kept it when c&p-ing.

Reply:
I think it's just a mistake that's been perpetuated. I've cleaned it out of the other test file in another pr